app/models/question.rb

```diff
@@ -2,9 +2,27 @@ class Question < ApplicationRecord
   include RailsAdminConf
 
   has_many :answers, dependent: :destroy, inverse_of: :question # the questions choices
+  has_many :user_questions_answers, dependent: :destroy
 
   accepts_nested_attributes_for :answers, allow_destroy: true
 
   validates :name, presence: true, uniqueness: true
+  validates_presence_of :category
+
+  enum category: [:daily, :weekly]
+
+  # It will filter the questions based on activeness and category i.e daily and weekly
+  scope :active, -> { where(active: true) }
+  scope :daily, -> { where(category: 'daily') }
```

Comment:
can we use enums here rather than hardcoding string ?

Reply:
Something like `where(category: :daily)` and `where(category: :weekly)`, since we have already declared enums I believe we can use them directly.